src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.UnitConverter/InputInterpreter.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System.Globalization;
+using System.Text.RegularExpressions;
+using UnitsNet;
+using System.Linq;
+
+namespace Community.PowerToys.Run.Plugin.UnitConverter
+{
+    public static class InputInterpreter
+    {
+        public static string[] RegexSplitter(string[] split) {
+            return Regex.Split(split[0], @"(?<=\d)(?![,.])(?=\D)|(?<=\D)(?<![,.])(?=\d)");
+        }
+
+        /// <summary>
+        /// Separates input like: "1ft in cm" to "1 ft in cm"
+        /// </summary>
+        /// <param name="split"></param>
+        public static void InputSpaceInserter(ref string[] split) {
+            if (split.Length != 3) {
+                return;
+            }
+
+            string[] parseInputWithoutSpace = Regex.Split(split[0], @"(?<=\d)(?![,.])(?=\D)|(?<=\D)(?<![,.])(?=\d)");
+
+            if (parseInputWithoutSpace.Length > 1) {
+                string[] firstEntryRemoved = split.Skip(1).ToArray();
+                string[] newSplit = new string[] { parseInputWithoutSpace[0], parseInputWithoutSpace[1] };
+
+                split = newSplit.Concat(firstEntryRemoved).ToArray();
+            }
+        }
+
+        /// <summary>
+        /// Replaces a split input array with shorthand feet/inch notation (1', 1'2" etc) to 'x foot in cm'. 
+        /// </summary>
+        /// <param name="split"></param>
+        /// <param name="culture"></param>
+        public static void ShorthandFeetInchHandler(ref string[] split, CultureInfo culture) {
+            if (!split[0].Contains('\'') && !split[0].Contains('\"')) {
```

Comment:
is stylecop / fxcop enabled here?  this should have been flagged as it is a Java / JS style brace, not c#